pokedex_async_redux/lib/utils/constants.dart

```diff
@@ -1,5 +1,36 @@
+import 'package:flutter/material.dart';
+
 const offsetPokemon = '0';
 const limitPokemon = '151';
 
-//TODO: This will be removed later; for testing PokemonDetails only
-const selectedPokemon = 'snorlax';
+///Pokemon Overview Page String Constants
+const imageUrlPrefix = 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/home/[index]';
+const imageUrlSuffix = '.png';
+const String pokemonOverviewTitle = 'Pokedex';
+const String type = 'Type';
+const String typePrimary = 'Type 1';
+const FontWeight textFontWeight = FontWeight.bold;
+
+///Pokemon Overview Page Number Constants
+const double defaultOne = 1.0;
+const int defaultTwo = 2;
+const double cardDefaultSize = 5.0;
+const double cardBorderRadius = 15.0;
+const double pokemonNameContainerHeight = 40.0;
+const double textPaddingVertical = 10.0;
+const double pokemonNameFontSize = 20.0;
+const double typeContainerHeight = 30.0;
+const double typePaddingHorizontal = 12.0;
+const double typeContainerBorderRadius = 15.0;
+const double typeTextFontSize = 16.0;
+const double typePrimaryTextFontSize = 12.0;
+const double typePrimaryBoxHeight = 20.0;
+const double typePrimaryBoxWidth = 80.0;
+const double typeBoxBorderRadius = 10.0;
+const double typeBoxTextFontSize = 12.0;
+const double pokemonImageSize = 150.0;
+
+/// //Pokemon Overview Page Color Constants
+const Color lightColor = Colors.white;
```

Comment:
u can create a new dart file for these color properties, u can name it as colors.dart 🙂